File: 30DaysOfCodingJAVA/Day_16_Exeptions.java
public class Solution {
    public static void main(String[] args) throws IOException {
        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));

        String S = bufferedReader.readLine();
        
        try{
            System.out.println(Integer.parseInt(S));
        }
        catch(Exception e){
            System.out.println("Bad String");
        }
        //finally{}
        bufferedReader.close();
    }
}

File: 30_days_of_coding/Day_16_Exeptions.cs
class Solution
{
    public static void Main(string[] args)
    {
        string S = Console.ReadLine();
        
        try{
            Console.WriteLine(Convert.ToInt32(S));
        }
        catch (FormatException e){
            Console.WriteLine("Bad String");
        }
        //finally{}
    }
}
